code/work_2.sh

```diff
@@ -0,0 +1,60 @@
+#!bin/bash
```

Comment:
shellcheck结果：
In ../2015-linux-public-icoy/code/work_2.sh line 38:
echo "The number of players between 20 and 30 years old is:$Number2,percentage:$Percent2%." >>$outfile 
^-- SC2129: Consider using { cmd1; cmd2; } >> file instead of individual redirects.


In ../2015-linux-public-icoy/code/work_2.sh line 47:
echo $(sed -e '1d' $file|awk -F '\t' '{if ( length($9)=='$longN' ) print $9}') >>$outfile
     ^-- SC2046: Quote this to prevent word splitting.
     ^-- SC2005: Useless echo? Instead of 'echo $(cmd)', just use 'cmd'.
                                                         ^-- SC2086: Double quote to prevent globbing and word splitting.


In ../2015-linux-public-icoy/code/work_2.sh line 50:
echo $(sed -e '1d' $file|awk -F '\t' '{if ( length($9)=='$shortN' ) print $9}') >>$outfile
     ^-- SC2046: Quote this to prevent word splitting.
     ^-- SC2005: Useless echo? Instead of 'echo $(cmd)', just use 'cmd'.
                                                         ^-- SC2086: Double quote to prevent globbing and word splitting.


In ../2015-linux-public-icoy/code/work_2.sh line 53:
echo $(sed -e '1d' $file|awk -F '\t' '{if ( $6=='$oldN') print $9}')  >>$outfile
     ^-- SC2046: Quote this to prevent word splitting.
     ^-- SC2005: Useless echo? Instead of 'echo $(cmd)', just use 'cmd'.
                                                 ^-- SC2086: Double quote to prevent globbing and word splitting.


In ../2015-linux-public-icoy/code/work_2.sh line 56:
echo $(sed -e '1d' $file|awk -F '\t' '{if ( $6=='$youngN') print $9}') >>$outfile
     ^-- SC2046: Quote this to prevent word splitting.
     ^-- SC2005: Useless echo? Instead of 'echo $(cmd)', just use 'cmd'.
                                                 ^-- SC2086: Double quote to prevent globbing and word splitting.